content/en/docs/setup/upgrade/canary/index.md

```diff
@@ -57,11 +57,14 @@ istio-sidecar-injector-canary   1          3m18s
 {{< /text >}}
 
 {{< warning >}}
-Due to [a bug](https://github.com/istio/istio/issues/28880) in the creation of the `ValidatingWebhookConfiguration` during install, initial installations of Istio __must not__ specify a revision. As a temporary workaround, for Istio resource validation to continue working after removing the non-revisioned Istio installation, the `ValidatingWebhookConfiguration` must be manually pointed to the revision that should handle validation.
+Due to [a bug](https://github.com/istio/istio/issues/28880) in the creation of the `ValidatingWebhookConfiguration` during install, initial installations of Istio __must not__ specify a revision. As a temporary workaround, for Istio resource validation to continue working after removing all revisioned Istio installations, the `istiod` service must be manually pointed to the revision that should handle validation.
```

Comment:
I think the text is wrong here, the workaround is for after you remove the non-revisioned install, not when you remove revisioned installs.

Reply:
Good catch thanks, the phrasing was indeed backwards. Fixed now